### DISCUSSION TITLE
Cannot edit repository description in GitHub mobile

### DISCUSSION THREAD

**Discussion by marckruzik** [community/community#43622] [Repositories]
When using GitHub mobile, on the repository main page, the "about" button is not displayed next to the description, so it is impossible to edit the repository description.

Here is a screenshot, with the orange circle showing where the "about" button could be displayed:

![IMG_20230108_140413.jpg](https://user-images.githubusercontent.com/51711029/211197591-92192e0c-e431-4a1a-965e-6241e614fd4a.jpg)

I can still see this button using "Desktop site" option on my mobile browser. But I think it should be possible to add this button on the mobile version as well.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @marckruzik ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

Similar to the feedback I provided in your last post, I've gone ahead and moved this to the correct category for you so that your feedback can get visibility. Thanks!

**Reply by marckruzik** [reply]
Thanks for your help, ghostinhershell!

**Comment by sebastienbaudry** [comment]
Same problem for me
